### DISCUSSION TITLE
Uno platform with grpc streaming

### DISCUSSION THREAD

**Discussion by MasterThreadInf** [unoplatform/uno#10331] [Q&A]
I was trying to create an example for Uno platform with grpc streaming. But at the moment , client side streaming and bidirectional streaming isn't working.  Is it made intentionally like that or am I missing something?

example solution: 
https://github.com/MasterThreadInf/UnoWithGrpcstreaming

**Comment by jeromelaban** [comment]
Thanks for the report. Can you explain what's not working? Do you have error messages? Which platform are you testing with?

**Reply by MasterThreadInf** [reply]
Currently testing only for wasm. There is no error messages but the call isn't working, meaning it doesn't comes in method where it is intented to ,but it goes to automatically created grpc method for e.g. clientside() in grpc instead of override method given by me. It must come in override .

**Reply by jeromelaban** [reply]
Do you see anything in the browser's output log, or in the network tab?

**Reply by MasterThreadInf** [reply]
unfortunately nothing

**Reply by jeromelaban** [reply]
When starting the Wasm head, I'm getting this:
```
dotnet.js:1 
       System.InvalidOperationException: 
Channel is configured with an HTTP transport doesn't support client-side load balancing or connectivity state tracking. 
The underlying HTTP transport must be a SocketsHttpHandler with no SocketsHttpHandler.ConnectCallback configured. 
The HTTP transport must be configured on the channel using GrpcChannelOptions.HttpHandler.
```

You may need to fix that first.
